pkgs/development/interpreters/luajit/default.nix

```diff
@@ -62,28 +62,33 @@ let
     else buildPackages.stdenv;
 
 in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "luajit";
   inherit version src;
 
   luaversion = "5.1";
 
+  LUA_PATH_PATTERN = [
+    "./?.lua"
+    "./share/lua/luajit-${finalAttrs.version}/?.lua"
+    "./share/lua/luajit-${finalAttrs.version}/?/init.lua"
+  ];
+
+  LUA_CPATH_PATTERN = [
+    "./?.so"
+    "./lib/lua/${finalAttrs.luaversion}/?.so"
+    "./lib/lua/${finalAttrs.luaversion}/loadall.so"
+  ];
```

Comment:
this feels like the opposite of DRY

also, the way these variables are named makes it sound like they are part of the lua specification instead of being a nix implementation detail.

Reply:
what is DRY ?

Reply:
don't repeat yourself

Reply:
resolved.